com.stansassets.plugins-dev-kit/Runtime/PackageSettings/LocalProjectSettings.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using UnityEngine;
+
+namespace StansAssets.Plugins
+{
+    [Serializable]
+    public class LocalProjectSettings
+    {
+        public LocalProjectSettings()
+        {
+            PackageName = "newPackage";
+            SettingsFileName = GetType().Name;
+        }
+        
+        public LocalProjectSettings(string packageName, string settingsFileName = null)
+        {
+            PackageName = packageName;
+            SettingsFileName = settingsFileName == null ? settingsFileName : GetType().Name;
+        }
```

Comment:
Why do we need it and where do we use it?
Plus seems like we should have `ILocalProjectSettings ` & `StansAssetsLocalProjectSettings` that holds basic path implementations like `$"{Application.dataPath}/{PackagesConfig.LibraryPath}/{PackageName}";` ets.